src/App.vue

```diff
@@ -80,11 +76,19 @@ export default class App extends Vue {
 
     private setRecipientAddress(address: string) {
         this.recipientAddress = address;
+        Vue.nextTick().then(() => {
+            const top = this.$children[1].$el.getBoundingClientRect().top;
+            window.scrollBy(0, top);
```

Comment:
I don't think this works as you expect. It scrolls _by_ the hight amount, so is a relative moment. But the `top` value you get is an absolute value, right? So it should rather use `scrollTo`. 

And now we come to the thing that I said, where the `behavior: 'smooth'` option is not backwards-compatible, because it can only be used with an options object to `scrollTo`. Which is not supported in Safari.

So I think you need to `try` the options syntax and if it `catch`es, use the legacy syntax.

Reply:
Additionally, I would prefer to use a `ref` for the element-to-scroll-to, instead of `children[0]`.

Reply:
> I don't think this works as you expect. It scrolls by the hight amount, so is a relative movement. But the top value you get is an absolute value, right? So it should rather use scrollTo.

Actually the `top` value i get is relative to the top of the screen. So `scrollBy` is appropriate here i think.

> And now we come to the thing that I said, where the behavior: 'smooth' option is not backwards-compatible, because it can only be used with an options object to scrollTo. Which is not supported in Safari.
> 
> So I think you need to try the options syntax and if it catches, use the legacy syntax.

Yes, you're right. I'll do that. I didn't paid that much attention to it because it was actually working on browser that were supposed to not support it.

> Additionally, I would prefer to use a ref for the element-to-scroll-to, instead of children[0].

Isn't it better to use `$children[1]` in this case if it's for a one time use only ? Or do you prefer to use `ref` because of maintainability ?